src/deepsparse/server/cli.py

```diff
@@ -206,6 +208,15 @@ def main(
        ...
     ```
     """
+
+    def _fetch_server(integration: str, config_path: str):
+        if integration == "local":
+            return DeepsparseServer(server_config=config_path)
+        elif integration == "sagemaker":
+            return SagemakerServer(server_config=config_path)
+        else:
```

Comment:
why do we default to open AI? rather than local or just raising error for unknown integration